test/test.js

```diff
@@ -1,6 +1,7 @@
 const request = require('supertest');
-const app = require('../app');
 const debug = require('debug')('memento:server');
```

Comment:
Even though we can't test that this is actually running at a given interval, we should be able to test the functions (fixLabelCounts, for instance). I think a simple test would be warranted, to make sure the query is right. There's already some code inside this file to wipe the db before/after a test; see "Test video prioritization".